src/main/java/org/bdgenomics/convert/ga4gh/BdgenomicsAlignmentRecordToGa4ghReadAlignment.java

```diff
@@ -0,0 +1,183 @@
+/**
+ * Licensed to Big Data Genomics (BDG) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The BDG licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bdgenomics.convert.ga4gh;
+
+import javax.annotation.concurrent.Immutable;
+
+import ga4gh.Common;
+import ga4gh.Reads;
+import org.bdgenomics.convert.AbstractConverter;
+import org.bdgenomics.convert.Converter;
+import org.bdgenomics.convert.ConversionException;
+import org.bdgenomics.convert.ConversionStringency;
+
+import org.bdgenomics.formats.avro.AlignmentRecord;
+import htsjdk.samtools.Cigar;
+import htsjdk.samtools.CigarOperator;
+import htsjdk.samtools.TextCigarCodec;
+import htsjdk.samtools.ValidationStringency;
+import htsjdk.samtools.CigarElement;
+
+
+
+import org.slf4j.Logger;
+
+import java.util.*;
```

Comment:
Prefer not to use star imports.  The imports should be sorted with java/javax first, then alphabetically.